### PR TITLE
Add/Refactoring IndexUtils #2

### LINKED ISSUE #2: Dump out tf-idf document vectors in pyserini
Is it possible to dump out tf-idf document vectors for retrieved documents in pyserini?

### PR DESCRIPTION
**Pull request by alirezamirzaeiyan**
Since DocVectorWeight enum is private in Anserini I couldn't port properly. This issue creates output files with wrong names(text.txt.docvector.(0-1).tar.gz). So it would be better to make DocVectorWeight public in IndexUtils.java. However, this piece of code works correctly.

### CODE REVIEW

File: pyserini/index/pyutils.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    '''
+
+    def __init__(self, index_dir):
+        self.object = JIndexUtils(JString(index_dir))
+
+    class DocVectorWeight(Enum):
+        NONE = 0
+        TF_IDF = 1
+
+    def dumpDocumentVectors(self, reqDocidsPath: str, weight: DocVectorWeight):
```
Comment (lintool) [inline pyserini/index/pyutils.py]: rename to Python conventions? i.e., `dump_document_vectors`?

**Comment by lintool** [comment]
> So it would be better to make DocVectorWeight public in IndexUtils.java

Yes, makes sense. Please send a PR in anserini directly?
Should we rename to `DocumentVectorWeight` while we're at it to be consistent?

File: pyserini/index/pyutils.py (hunk 1 above)
Comment (alirezamirzaeiyan) [reply pyserini/index/pyutils.py]: Yes :)

**Comment by alirezamirzaeiyan** [comment]
Yes, I'll send a PR in anserini asap. It would be very great to be consistent, by the way, should I rename to DocumentVectorWeight in anserini?

**Comment by lintool** [comment]
> should I rename to DocumentVectorWeight in anserini?

yes please.

**Comment by lintool** [comment]
It would be even more amazing if you could write a test case!

**Comment by alirezamirzaeiyan** [comment]
I've fixed enum problem by a PR in anserini and another commit in pyserini.

**Comment by alirezamirzaeiyan** [comment]
> t would be even more amazing if you could write a test case!

Do you mean to write a test case for dumpDocumentVectors in anserini?

**Comment by lintool** [comment]
Yes

**Pull request merged**